app/Photo.php

```diff
@@ -25,6 +26,12 @@ public static function getLastRecord()
         return self::orderBy('id', 'desc')->first();
     }
 
+    public static function getLastRowByTime($timeInMinutes)
```

Comment:
getLast($since)

Reply:
Changed

Reply:
I'm sorry, this method is unused in the code. It was only used in the first test proposal.